tasks/config.ts

```diff
@@ -131,117 +127,129 @@ export function load() {
       }
     });
   });
-}
-
-async function getPairsOraclesAndSymbols(hre: HardhatRuntimeEnvironment, deployer: SignerWithAddress) {
-  const factory = await getUniswapFactory(hre);
-  const pools = await getPools(hre);
-
-  const pairInfos: {
-    pair: { address: string; token0: string; token1: string; token0Symbol: string; token1Symbol: string };
-    pairOracle: { pairAddress: string; oracleAddress: string };
-    symbol: string;
-  }[] = [];
-  for (const poolPair of pools) {
-    let pairSymbol = getPoolKey(poolPair[0].token.address, poolPair[1].token.address, poolPair[0].name, poolPair[1].name);
-    const pairAddress = await factory.getPair(poolPair[0].token.address, poolPair[1].token.address);
-    const oracleAddress = (await hre.ethers.getContract(`UniswapPairOracle_${pairSymbol}`, deployer)).address;
-
-    if (hre.network.name.toLowerCase() == "rsk") {
-      pairSymbol = pairSymbol.replace("ETH", "RBTC").replace("WBTC", "ETHs");
-    }
 
-    pairInfos.push({
-      pair: {
-        address: pairAddress,
-        token0: poolPair[0].token.address,
-        token1: poolPair[1].token.address,
-        token0Symbol: poolPair[0].name,
-        token1Symbol: poolPair[1].name
-      },
-      pairOracle: { pairAddress: pairAddress, oracleAddress: oracleAddress },
-      symbol: pairSymbol
+  async function getPriceFeedsConfig(hre: HardhatRuntimeEnvironment, deployer: SignerWithAddress) {
```

Comment:
related to my question here: https://github.com/BlindexDAO/backend/pull/48#discussion_r799548228

Let's discuss it there

Reply:
1. I have changed the structure because array allows for more generalized approach and makes it easier to add new price feeds (which I guess we will add along with new bdstables). Furthermore, the array has 5 elements and won't grow much more so find() execution cost is negligible.
2. It can be - BtcToEthOracleChinlink (mainnet price feed) doesn't export any decimals function

I didn't notice some price feeds expose getDecimals() instead of decimals() so I have changed the implementation to use any of these two methods (whichever is available).

Reply:
1. 👍 
2. Let's continue the discussion here - https://github.com/BlindexDAO/BlinDEX/pull/115/files#r801317207